app/src/auth.py

```diff
@@ -51,7 +51,16 @@ def register():
         #Access the data submitted to form
         email = request.form.get("email")
         username = request.form.get("username")
+
+        #Check the Strength of the password
         password = request.form.get("password")
+        result = check_password_strength(password)
+        if not result:
+            flash("Password is not strong enough")
+        else:
+            flash("Password is not strong enough. Here are some suggestions: " + ", ".join(result))
+            return redirect(url_for("register"))
```

Comment:
If the password is strong enough, result will be None. So `if not result` is the situation where the password is strong enough, and `else` is when the password is not strong enough.

Reply:
Changed the message to display "Password is strong enough".